AccessHandle.md

```diff
@@ -22,7 +22,7 @@
   - [New data access surface](#new-data-access-surface)
   - [Locking semantics](#locking-semantics)
 - [Open Questions](#open-questions)
-  - [Naming](#naming)
+  - [Exposing AccessHandles on all filesystems](#exposing-accesshandles-on-all-filesystems)
```

Comment:
Not really part of this PR, but you might consider finding a better title here. It's not clear what "filesystems" refers to in this context. Did you mean "file handles"?